sentry.py

```
@@ -0,0 +1,101 @@
+username_md5 = hashlib.md5(username.encode()).hexdigest()
+print("Username MD5: ", username_md5)
+sentry_sdk.set_tag("username.md5", username_md5)
+
+
+cache_dir = os.path.expanduser("~/.cache/partcad")
+os.makedirs(cache_dir, exist_ok=True)
+cache_file = os.path.join(cache_dir, "id")
+
+# Check if the UID is already cached
+if os.path.exists(cache_file):
+    with open(cache_file, "r") as f:
```

Ruff (0.8.2)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)